main.go

```diff
@@ -134,7 +140,10 @@ func main() {
 		Port:                   9443,
 		HealthProbeBindAddress: probeAddr,
 		LeaderElection:         enableLeaderElection,
-		LeaderElectionID:       "8b4defce.openshift.io",
```

Comment:
I believe this was created by operator sdk , whats the motivation of changing the ID?

Reply:
better meaningful identifier ?

Reply:
I think the change is valid, from docs https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/manager#Options does not seem to be a standard for it